### DISCUSSION TITLE
Outreachy applicant

### DISCUSSION THREAD

**Discussion by donaldte** [LesFruitsDefendus/saskatoon-ng#127] [Outreachy]
#Outreachy Applicant 
Good moring to you my name is Tedom Donald studying computer engineering in Cameroon.

and I am an Outreachy applicant I have chosen to contribute to this magnificent project in order to be accepted as an intern at Outreachy.

Thanks for the opportunity

**Comment by tristanlatr** [comment]
Hello @donaldte,

Glad to see that you find this project magnificent, look forward for your contibutions. 

Tristan

**Comment by aryabyte21** [comment]


**Reply by tristanlatr** [reply]
Hello @AryaArunB,

Glad to see your interested into Saskatoon.

Look forward for your contribution.

**Comment by angelnagaba** [comment]
Hello My name is Angel Nagaba studying software Engineering in Uganda. I have chosen to contribute to this project as an applicant for outreachy May 2022. 

I am excited for this opportunity. 
Thank you

**Reply by tristanlatr** [reply]
Hello @angelnagaba,

Thanks for your interest into Saskatoon, I look forward for your contributions. 

Tristan

**Comment by effulgentstar** [comment]
Hello, My name is Lucy Ajah from Nigeria, I would like to make contributions to this project as an applicant for Outreachy may 2022

**Reply by tristanlatr** [reply]
Hi @effulgentstar,

Thanks for your interest into Saskatoon.

Look forward for your contributions!

**Comment by pytechlady** [comment]
Hello, my name is Adebimpe from Lagos, Nigeria. I am an applicant for Outreach May 2022 internship and I will like to contribute to this project.

**Reply by tristanlatr** [reply]
Hello @pytechlady,

Look forward for your contribution!

Talk to you later,

**Comment by HASSAN1A** [comment]
Happy to be here my name is Hassan Juma really interested to contribute to the project.

**Reply by tristanlatr** [reply]
Hello @HASSAN1A,

Thanks for your interest into Saskatoon, I look forward to your contributions!

Tristan

**Comment by ajaypheelz** [comment]
Hello , its a pleasure to be here , my name is David and I  am looking foward to contribute on this FOSS project

**Reply by tristanlatr** [reply]
Hello @ajaypheelz,

Thanks for you interest into Saskatoon, look forward for your contribution.

Tristan,
